src/pages/Reservations.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchReservations } from '../redux/xers/reservationXer';
+import { returnCurrentUser } from '../services/cookie';
+
+import '../assets/App.css';
+import '../assets/reservations.css';
+
+const Reservations = () => {
```

Comment:
- [ ] There is a bug on the "My reservations" section where the app shows the reservations for all other users, instead of just my own reservations (in this case, I'm `user_id: 4`):

![image](https://user-images.githubusercontent.com/57421823/206111990-facc5238-ab8f-45f0-bfae-b7164b0432c3.png)

Kindly fix this bug so your app works smoothly as intended. 💪